README.md

```diff
@@ -83,13 +103,15 @@ Finally, since the fusion of carrier algebras occurs as a result of the selectio
 Like other effect systems, effects are performed in a `Monad` extended with operations relating to the effect. In `fused-effects`, this is done by means of a `Has` constraint to require the effect’s presence in a _signature_, and to relate the signature to the _carrier_ you’re computing in. For example, to use a `State` effect managing a `String`, one would write:
 
 ```haskell
-action :: Has (State String) sig m => m ()
+action1 :: Has (State String) sig m => m ()
+action1 = get >>= \ s -> put ("hello, " ++ s)
```

Comment:
Step 3: we flesh out any definitions which we’d written just as type signatures before.